src/store/item-store.js

```diff
@@ -34,4 +31,8 @@ store.registerHandler('CLEAR_ALL_DATA', () => {
     store.emitChange();
 });
 
+store.registerHandler('ITEMS_RECEIVED', data => {
+    item = data.items;
```

Comment:
I guess it could? I did it this way because I thought it would be more confusing if there was a variable named item and items. Like what's the difference at that point and why would we need to make server calls if items contains all items?